src/redux/slices/entryManager/entryManager.slice.ts

```diff
@@ -48,6 +48,14 @@ export const getAllEntries = (state: StoreState) => state.entryManager.entries;
 
 export const getVisibleEntries = (state: StoreState) => state.entryManager.visibleEntries;
 
+export const getVisibleEntriesByLikes = (state: StoreState) => {
+    const entries = Object.values(state.entryManager.visibleEntries); // get all visible entries
+    const likes = entries.map((entry) => state.entryManager.entries[entry]?.likes); // get all likes
```

Comment:
No need for nullish coalescing here:
```suggestion
    const likes = entries.map((entry) => state.entryManager.entries[entry].likes); // get all likes
```